misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -80,6 +92,7 @@ public CoverageReport(Module module) throws IOException {
      * @throws IOException when file operations are failed
      */
     public ModuleCoverage generateReport(JarResolver jarResolver) throws IOException {
+
```

Comment:
Lets remove the new line right after the function as well. Apply this for the other functions as well